### PR TITLE
CLOUD-310 Documentation update

### PR DESCRIPTION
**Pull request by cap1984**
Implementing doc update after verification

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/percona-server-mongodb-operator?pullRequest=225) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-server-mongodb-operator?pullRequest=225) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-server-mongodb-operator?pullRequest=225) it.</sub>

File: source/private.rst [hunk 1]
```diff
@@ -21,10 +21,9 @@ backups involves following steps:
            --set accessKey=some-access-key \
            --set secretKey=some-secret-key \
            --set service.type=ClusterIP \
-           --set configPath=/tmp/.minio/ \
+           --set configPath='~/.minio' \
```
Comment (delgod) [inline source/private.rst]: is this path works fine on OpenShift?

File: source/private.rst [hunk 2]
```diff
@@ -35,15 +34,19 @@ backups involves following steps:
    for backups. Otherwise, this setting may be omitted. You may also notice the
    ``MINIO_REGION`` value which is may not be used within a private
    cloud. Use the same region value here and on later steps
-   (``us-east-1`` is a good default choice).
+   (``us-west-1`` is a good default choice).
```
Comment (delgod) [inline source/private.rst]: us-east-1 is a typical default, why is `us-west-1` better?

File: source/openshift.rst [hunk 3]
```diff
@@ -83,7 +83,7 @@ Install Percona server for MongoDB on OpenShift
 
    More details about secrets can be found in Users.
 
-5. Now you need to `prepare certificates for TLS security <TLS.html>`_ and apply them with the following command:
+5. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/openshift.html>`_ if it is not up and running yet and apply certificates as openshift secrets with the following command:
```
Comment (delgod) [inline source/openshift.rst]:
```suggestion
5. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/openshift.html>`_ if it is not up and running yet then generate and apply certificates as secrets according to `TLS document <TLS.html>`
```

File: source/private.rst (hunk 1 above)
Comment (cap1984) [reply source/private.rst]: Yeap. I made such a change in order to make this command work on Win + git bash. '/tmp/.minio' path is reconstructed into something like 'C:\Users\...' by git bash which leads to startup failure.

File: source/private.rst (hunk 2 above)
Comment (cap1984) [reply source/private.rst]: It's more like synchronization between our doc file and cr.yaml. cr.yaml by default comes with us-west-1 region, which I can't change in this PR.

File: source/openshift.rst (hunk 3 above)
Comment (cap1984) [reply source/openshift.rst]: agree

File: source/private.rst (hunk 1 above)
Comment (delgod) [reply source/private.rst]: please test it on real OpenShift

File: source/openshift.rst [hunk 4]
```diff
@@ -83,7 +83,7 @@ Install Percona server for MongoDB on OpenShift
 
    More details about secrets can be found in Users.
 
-5. Now you need to `prepare certificates for TLS security <TLS.html>`_ and apply them with the following command:
+5. Install `cert-manager <https://docs.cert-manager.io/en/release-0.8/getting-started/install/openshift.html>`_ if it is not up and running yet then generate and apply certificates as secrets according to `TLS document <TLS.html>`:
```
Comment (delgod) [inline source/openshift.rst]: could we put line 92 before line 88?

File: source/private.rst (hunk 2 above)
Comment (delgod) [reply source/private.rst]: Could we just change CR, not doc in this case? I prefer `us-east-1`

Comment (cap1984) [reply source/private.rst]: ok

File: source/openshift.rst (hunk 4 above)
Comment (cap1984) [reply source/openshift.rst]: ok

**Comment by cap1984** [comment]
configPath has been reverted

**Pull request merged**